client/src/index.jsx

```diff
@@ -1,6 +1,6 @@
 /* eslint-disable import/extensions */
 import React from 'react';
 import ReactDOM from 'react-dom';
-import App from './App.jsx';
+import App from './App';
```

Comment:
That webconfig section saved us some typing. :)

Reply:
@rickboot 🤩